README.md

```diff
@@ -1,86 +1,257 @@
+
+[<img width="12px" src="https://user-images.githubusercontent.com/26427004/104119722-9033c600-5385-11eb-9d57-4c376862fd36.png"> **SOFTPEDIA EDITOR'S PICK**](https://www.softpedia.com/get/System/Launchers-Shutdown-Tools/Flow-Launcher.shtml)
+
+## 🎉 New Features in 1.9 
+
+- All New Design. New Themes, New Setting Window. Animation & Sound Effect, Color Scheme aka Dark Mode.
+- New Plugins, Plugin Store, Game Mode, Wizard window
+
+<a href="https://github.com/Flow-Launcher/Flow.Launcher/releases/tag/v1.9.0">Full changelog</a>
 
 ---
 
 <h4 align="center">
-  <a href="#Features">Features</a> •
-  <a href="#Getting-Started">Getting Started</a> •
+  <a href="#Getting-Started">Getting Started</a> •  <a href="#Features">Features</a> • <a href="#Plugins">Plugins</a> •
+       <a href="#Hotkeys">Hotkeys</a> •
   <a href="#QuestionsSuggestions">Questions/Suggestions</a> •
   <a href="#Development">Development</a> •
-  <a href="https://flow-launcher.github.io/docs">Documentation</a>
+  <a href="https://flow-launcher.github.io/docs">Docs</a>
 </h4>
 
+
 ---
 
+## 🚗 Getting Started
+
+### Installation
+
+| [Windows 7+ installer](https://github.com/Flow-Launcher/Flow.Launcher/releases/latest/download/Flow-Launcher-Setup.exe) | [Portable](https://github.com/Flow-Launcher/Flow.Launcher/releases/latest/download/Flow-Launcher-Portable.zip) | `WinGet install "Flow Launcher"` |
+| :----------------------------------------------------------: | :----------------------------------------------------------: | :------------------------------: |
+
+> Windows may complain about security due to code not being signed, this will be completed at a later stage. If you downloaded from this repo, you are good to continue the set up. 
+
+And You can download <a href="https://github.com/Flow-Launcher/Flow.Launcher/discussions/640" target="_blank">early access version</a>.
+
+
 ## Features
 
-![The Flow](https://user-images.githubusercontent.com/26427004/82151677-fa9c7100-989f-11ea-9143-81de60aaf07d.gif)
+### Applications & Files
+
+<img src="https://user-images.githubusercontent.com/6903107/144517107-efd83586-9084-46e0-b7dc-79a6748ea936.png" width="500">
 
-- Search everything from applications, files, bookmarks, YouTube, Twitter and more. All from the comfort of your keyboard without ever touching the mouse.
+- Flow searches files and contents via Windows Index Search
 - Search for file contents.
-- Do mathematical calculations and copy the result to clipboard. 
+
+<img src="https://user-images.githubusercontent.com/6903107/144517293-622844c8-fe4b-4ca6-82b0-4aa6feca4ddd.png" width="400">
+
 - Support search using environment variable paths.
+
+### Web Search & Open URL
+
+<img src="https://user-images.githubusercontent.com/6903107/144517502-5325de01-d0d9-4c2e-aafb-33c3f5d82f81.png" width="400">
+<img src="https://user-images.githubusercontent.com/6903107/144517508-b2b78aaa-9bdb-4e65-85fd-d659f4157c80.png" width="400">
```

Comment:
this action keyword is no longer `g`, it's now global, screenshot need to be updated sorry

Reply:
nvm. ok. I'll change it.